src/series/column.js

```diff
@@ -0,0 +1,120 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.series.column = function() {
+
+        var decorate = fc.util.fn.noop,
+            xScale = d3.time.scale(),
+            yScale = d3.scale.linear(),
+            x1Value = function(d, i) { return d.close; },
+            yValue = function(d, i) { return d.date; },
+            x0Value = d3.functor(0),
+            columnWidth = fc.util.fractionalBarWidth(0.75);
+
+        var dataJoin = fc.util.dataJoin()
+            .selector('g.column')
+            .element('g')
+            .attr('class', 'column');
+
+        var yValueScaled = function(d, i) { return yScale(yValue(d, i)); };
+
+        var column = function(selection) {
+            selection.each(function(data, index) {
+
+                var filteredData = data.filter(function(d, i) {
+                    return x0Value(d, i) !== undefined &&
+                        x1Value(d, i) !== undefined &&
+                        yValue(d, i) !== undefined;
+                });
+
+                var g = dataJoin(this, filteredData);
+
+                var height = columnWidth(filteredData.map(yValueScaled));
```

Comment:
I'm really confused about the orientation in this line, are the variables named appropriately?